api/scripts/team-acces/update-fwb-authentication-complements.js

```diff
@@ -0,0 +1,89 @@
+import url from 'url';
+
+import _ from 'lodash';
+import bluebird from 'bluebird';
+
+import { disconnect } from '../../db/knex-database-connection.js';
+import { checkCsvHeader, parseCsvWithHeader } from '../helpers/csvHelpers.js';
+import { usecases } from '../../lib/domain/usecases/index.js';
+import { FWB } from '../../lib/domain/constants/oidc-identity-providers.js';
+
+const MODULE_PATH = url.fileURLToPath(import.meta.url);
+const IS_LAUNCHED_FROM_CLI = process.argv[1] === MODULE_PATH;
+const TIMER_NAME = '';
+const REQUIRED_FIELD_NAMES = ['SUB', 'Firstname', 'Lastname', 'EmployeeNumber', 'Population'];
```

Comment:
Est-ce que l'on garde les noms de colonnes suivantes ou on les modifie ? Ce qui signifie modifier lorsque le fichier sera fournie par le partenaire.

```suggestion
const REQUIRED_FIELD_NAMES = ['sub', 'firstName', 'lastName', 'employeeNumber', 'population'];
```

Reply:
A moins qu'il y ait une convention sur le nommage des noms chez Pix, plutôt laisser non ? Comme c'est du csv, on peut avoir des noms en camelCase, PascalCase, en plusieurs mots  😸

Reply:
J'ai fait la modif  (pour être raccord avec le fichier de test)